app/services/catalog/create_approval_request.rb

```diff
@@ -2,9 +2,13 @@ module Catalog
   class CreateApprovalRequest
     attr_reader :order
 
-    def initialize(task)
+    def initialize(task: nil, order_id: nil)
```

Comment:
With no defaults we may run into a situation where both are true or non are true. Can the code handle either of those situations